https://github.com/libproxy/libproxy/pull/180

From 54a08b73f4a9bc43356dd50fe91942b4270bfe10 Mon Sep 17 00:00:00 2001
From: Matt Turner <mattst88@gmail.com>
Date: Tue, 21 Jun 2022 00:08:57 -0400
Subject: [PATCH] Fix building without duktape

Otherwise configuring fails with

CMake Error at libproxy/cmake/modules.cmk:26 (math):
  math cannot parse the expression: "0+0+0+": syntax error, unexpected end of
  file (6).
Call Stack (most recent call first):
  libproxy/CMakeLists.txt:14 (include)
---
 libproxy/cmake/modules/pacrunner_duktape.cmk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libproxy/cmake/modules/pacrunner_duktape.cmk b/libproxy/cmake/modules/pacrunner_duktape.cmk
index e7918a8..a468125 100644
--- a/libproxy/cmake/modules/pacrunner_duktape.cmk
+++ b/libproxy/cmake/modules/pacrunner_duktape.cmk
@@ -17,5 +17,7 @@ else()
       link_directories(${DUKTAPE_LIBRARIES})
       link_libraries(duktape)
     endif()
+  else()
+    set(DUKTAPE_FOUND 0)
   endif()
 endif()
-- 
2.35.1

